README.md

```diff
@@ -1,5 +1,10 @@
 # DevOps Apprenticeship: Project Exercise
 
+## This is for scripting for Automating
```

Comment:
Where's the script (it's not listed in your description)?

Reply:
This has been changed on Readme